services/src/test/scala/cromwell/pipeline/service/AggregationServiceTest.scala

```diff
@@ -60,10 +60,10 @@ class AggregationServiceTest extends AsyncWordSpec with Matchers with MockitoSug
       "should return exception if project not found" taggedAs Service in {
         when(projectService.getUserProjectById(projectId, userId)).thenReturn(Future.failed(NotFound()))
         when(projectConfigurationService.getLastByProjectId(projectId, userId))
-          .thenReturn(Future.successful(Some(projectConfigurations)))
+          .thenReturn(Future.successful(projectConfigurations))
 
         aggregatorService.aggregate(run).failed.map {
-          _ should have.message("Project not found")
+          _ should have.message("Configuration not found")
```

Comment:
That doesn't make sense to me. `projectConfigurationService.getLastByProjectId` succeeds with `projectConfigurations`, why do we expect `Configuration not found` message here? More importantly, why does this test work?